.github/CODEOWNERS

```diff
@@ -248,7 +248,7 @@ packages/react-components/react-calendar-compat @microsoft/cxe-red @sopranopillo
 packages/react-components/react-infolabel @microsoft/cxe-red @sopranopillow
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31112&commitId=b42287a951a3f90d3c151cd8201a53f5590b64be&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>